src/main/frontend/components/plugins.cljs

```diff
@@ -355,27 +355,35 @@
       (ui/dropdown-with-links
         (fn [{:keys [toggle-fn]}]
           (ui/button
-            [:span (ui/icon "arrows-sort") ""]
-            :class "sort-by"
+            [:span (ui/icon "arrows-sort")]
+            :class (str (when-not (contains? #{:enabled :downloads} sort-by) "picked ") "sort-by")
             :on-click toggle-fn
             :intent "link"))
         (let [aim-icon #(if (= sort-by %) "check" "circle")]
           (if market?
-            [{:title   "Downloads (Desc)"
+            [{:title   (t :plugin/downloads)
               :options {:on-click #(reset! *sort-by :downloads)}
               :icon    (ui/icon (aim-icon :downloads))}
 
-             {:title   "Stars (Desc)"
+             {:title   (t :plugin/stars)
               :options {:on-click #(reset! *sort-by :stars)}
               :icon    (ui/icon (aim-icon :stars))}
 
-             {:title   "Title (A - Z)"
+             {:title   (str (t :plugin/title) " (A - Z)")
               :options {:on-click #(reset! *sort-by :letters)}
               :icon    (ui/icon (aim-icon :letters))}]
 
             [{:title   (t :plugin/enabled)
               :options {:on-click #(reset! *sort-by :enabled)}
-              :icon    (ui/icon (aim-icon :enabled))}]))
+              :icon    (ui/icon (aim-icon :enabled))}
+
+             {:title   (t :plugin/disabled)
```

Comment:
This might be misleading.
It's not a filter, it's a sort-by.

Reply:
I also think it's a little bit. This is a forced sort-by (retains all the list contents), otherwise the icon will be replaced, which is inconsistent with the Marketplace. 

**let's go with the filter behavior here!**